harness/temporalHelpers.js

```diff
@@ -28,6 +28,17 @@ var TemporalHelpers = {
     assert.sameValue(duration.nanoseconds, nanoseconds, `${description} nanoseconds result`);
   },
 
+  /*
```

Comment:
This helper is added in https://github.com/tc39/test262/commit/11621016ccd76e521a22e1aeef3c437e67d9553f and is repeated here becuse these tests make use of it.